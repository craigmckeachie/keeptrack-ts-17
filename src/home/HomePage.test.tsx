import React from 'react';
import { render, screen } from '@testing-library/react';
import HomePage from './HomePage';
import renderer from 'react-test-renderer';

describe('<HomePage />', () => {
  test('renders home heading', () => {
    render(<HomePage />);
    expect(screen.getByRole('heading')).toHaveTextContent('Home');
  });

  test('snapshot', () => {
    const tree = renderer.create(<HomePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
