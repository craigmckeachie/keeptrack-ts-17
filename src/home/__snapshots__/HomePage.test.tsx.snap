// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> snapshot 1`] = `
<h2>
  Home
</h2>
`;
